packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -934,6 +987,21 @@ export default createRule<Options, MessageIds>({
       let memberTypes: MemberType[] | string | undefined;
       let optionalityOrder: OptionalityOrder | undefined;
 
+      /**
+       * It runs an alphabetic sort on the groups of the members of the class in the source code.
+       * @param memberSet The members in the class of the source code on which the grouping operation will be performed.
+       */
+      const checkAlphaSortForAllMembers = (memberSet: Member[]): undefined => {
+        const hasAlphaSort = !!(order && order !== 'as-written');
+        if (hasAlphaSort && Array.isArray(memberTypes)) {
+          groupMembersByType(memberSet, memberTypes, supportsModifiers).forEach(
+            groupMember => {
```

Comment:
[Naming] "Group member" is singular. But then this is passed to `checkAlphaSort`, which calls it "members" (plural). It's a little confusing to have the style of name for something change when it's used.

How about, maybe?:

```suggestion
            members => {
```

...or failing that, `groupMembers`?